lib/scala/kg/src/main/scala/formats/kg/kgtk/KgtkTsvReader.scala

```diff
@@ -0,0 +1,71 @@
+package formats.kg.kgtk
+
+import formats.CsvReader
+import com.github.tototoshi.csv.{CSVReader, TSVFormat}
+import models.kg.KgEdge
+import models.kg.KgNode
+import org.slf4j.LoggerFactory
+import java.nio.file.Path
+import java.io.InputStream
+
+final class KgtkTsvReader(csvReader: CSVReader) extends CsvReader[Tuple3[KgEdge, KgNode, KgNode]](csvReader) {
+  private final val LIST_DELIM = "|";
+
+  private val logger = LoggerFactory.getLogger(getClass)
+
+  def iterator: Iterator[Tuple3[KgEdge, KgNode, KgNode]] =
+    csvReader.iteratorWithHeaders.map(row => {
```

Comment:
The more idiomatic syntax is to use spaces. Something like

`csvReader iteratorWithHeaders map { row => ... }`

Reply:
Gotcha, I just tried it but idea is complaining that `iteratorWithHeaders does not take parameters`?